### PR TITLE
Add FreeBSD installation instructions.

### PR DESCRIPTION
**Pull request by arrowd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ git config --global alias.b brunch
 `git-brunch` is in the [AUR](https://aur.archlinux.org/packages/git-brunch).
 Install it with e.g. `yay -S git-brunch` or `pamac install git-brunch`.
 
+### FreeBSD
+
+`git-branch` can be installed from the official package repository with `pkg install hs-git-brunch`.
```
Comment (andys8) [inline README.md]:
```suggestion
`git-brunch` can be installed from the official package repository with `pkg install hs-git-brunch`.
```

**Review by andys8 — CHANGES_REQUESTED** [review]

**Comment by andys8** [comment]
Awesome. Did you add it or are haskell/stack packages automatically available?

There is a typo. Then it's good to go. Thank you.

**Comment by arrowd** [comment]
> Awesome. Did you add it or are haskell/stack packages automatically available?

Yes, I just added it: https://cgit.FreeBSD.org/ports/commit/?id=71a8292d43f45dd6ac593d0008da87b4263fe9e2

Is it possible for you to squash the two commits from the web interface, or should I do it?

**Comment by andys8** [comment]
Awesome. Thank you. I'll squash via github.

**Review by andys8 — APPROVED** [review]

**Pull request merged**